test/unit/spec/v2/view-builder/utils/ChallengeViewUtil_spec.js

```diff
@@ -19,7 +19,8 @@ describe('v2/utils/ChallengeViewUtil', function() {
       'challengeMethod': Enums.LOOPBACK_CHALLENGE,
       'domain': 'test_domain',
       'ports': [12345, 22222],
-      'challengerequest': 'abcdfg12345'
+      'challengerequest': 'abcdfg12345',
+      'probeTimeoutMillis': 101
```

Comment:
Why is this set to 101 instead of 100?

Reply:
Oh, seems like the test below just makes sure the request gets called with that value. Can we set this to 100 so it's consistent with the value we decided on?

Reply:
Value here is part of a purely client side unit test. Actual value does not appear significant - just needs to be non-null.

Reply:
I would still keep it consistent to `100` so that it doesn't confuses others

Reply:
Done!